hack/local-up-cluster.sh

```diff
@@ -940,6 +943,18 @@ function start_csi_snapshotter {
     fi
 }
 
+function start_volume_data_source_validator {
+    if [[ "${ENABLE_VOLUME_DATA_SOURCE_VALIDATOR}" = true ]]; then
+        echo "Creating volume data source validator"
+        ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f "${KUBE_ROOT}/cluster/addons/any-volume-datasource/crd/populator.storage.k8s.io_volumepopulators.yaml"
+        ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f "${KUBE_ROOT}/cluster/addons/any-volume-datasource/crd/hello-populator-crd.yaml"
```

Comment:
The Hello CRDs should be installed as part of the test (like what you did before).
Volume Populator CRDs should be installed by the cluster as what you have here.

Reply:
Thank you for your comment. It seems that we need to check [kube-master-configuration.log](https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/108664/pull-kubernetes-e2e-gce-alpha-features/1506685033644036096/artifacts/e2e-d4b68bf607-cc220-master/kube-master-configuration.log) for the addon's setup logs. And it seems failing.

```
Mar 23 18:07:52.448034 e2e-d4b68bf607-cc220-master configure-helper.sh[1288]: error: the server doesn't have a resource type "VolumePopulator"
Mar 23 18:07:52.749176 e2e-d4b68bf607-cc220-master configure-helper.sh[1288]: No resources found in default namespace.
Mar 23 18:07:52.750579 e2e-d4b68bf607-cc220-master configure-helper.sh[1288]: Wait until volume snapshot RBAC rules are installed
Mar 23 18:07:52.998490 e2e-d4b68bf607-cc220-master configure-helper.sh[1288]: NAME                              ROLE                                            AGE
Mar 23 18:07:52.998995 e2e-d4b68bf607-cc220-master configure-helper.sh[1288]: volume-snapshot-controller-role   ClusterRole/volume-snapshot-controller-runner   9s
Mar 23 18:07:53.000741 e2e-d4b68bf607-cc220-master configure-helper.sh[1288]: Wait until volume snapshot controller is installed
Mar 23 18:07:53.239418 e2e-d4b68bf607-cc220-master configure-helper.sh[1288]: Error from server (NotFound): statefulsets.apps "volume-snapshot-controller" not found
Mar 23 18:08:02.766193 e2e-d4b68bf607-cc220-master configure-helper.sh[1288]: error: the server doesn't have a resource type "VolumePopulator"
```

I added `addonmanager.kubernetes.io/mode=Reconcile` labels to all the VolumePopulator related resources, which looks like the biggest difference from snapshot ones, and fixed the check logics. And let me continue debugging.